src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -978,4 +981,78 @@ public void testTestFilterCollectionByExpressions(Set<String> values, Collection
         Set<String> actual = Utils.filterCollectionByExpressions(values, filters, exactMatch);
         Assert.assertEquals(actual, expected);
     }
+
+    @DataProvider(name = "concatMultipleByteArraysData")
+    public Object[][] concatMultipleByteArraysData() {
+        return new Object[][]{
+                // expected, arrays
+                new Object[]{ ArrayUtils.EMPTY_BYTE_ARRAY, new byte[] { } },
+                new Object[]{ new byte[] {100, 100}, new byte[] {100, 100} },
+                new Object[]{ new byte[] {100, 100}, new byte[] {100, 100}, new byte[] { } },
+                new Object[]{ ArrayUtils.EMPTY_BYTE_ARRAY, new byte[] { }, new byte[] { } },
+                new Object[]{ new byte[] {100, 100, 100, 100}, new byte[] {100, 100}, new byte[] {100, 100} },
+        };
```

Comment:
Comments on the test cases for the varargs version of concat():

- Calling concat() with 2 byte arrays actually calls the two-arg specialization of concat(), not the varargs version, so none of your test cases here should have 2 input arrays. Call concat() with 0, 1, or 3+ arrays to test the varargs version.
- The values inside your arrays should not all be the same (100), otherwise you are not actually testing whether the method is preserving the original order of the values correctly during concatenation.